src/asm/fstack.cpp

```diff
@@ -309,6 +310,20 @@ void fstk_RunInclude(std::string const &path, bool preInclude) {
 		fatalerror("Failed to set up lexer for file include\n");
 }
 
+std::set<std::string> includedOnce;
+
+void fstk_RunIncludeOnce(std::string const &path) {
+	if (includedOnce.find(path) != includedOnce.end()) {
+		if (verbose) {
+			printf("File '%s' already INCLUDE_ONCE'd, skipping", path.c_str());
+		}
+		return;
+	}
+
+	includedOnce.insert(path);
```

Comment:
Regular `INCLUDE` should also count as being included at least once.

Reply:
Inserting into the list of included files has been moved to the `fstk_RunInclude` function, in order for both types of includes to work.